README.md

```diff
@@ -30,75 +50,76 @@ The following parameters are optional
 - `backup_retention_days` number of days to retain a backup. Default is 35.
 - `georedundant_backup` specifies whether to use geo-redundant backup over local. Default is "Enabled".
 
-### Access to databases
+### Output
+
+The following variables are provided by the module for use in other modules
 
-Databases are restricted to access from specific subnets, these can be updated [here](https://github.com/hmcts/cnp-database-subnet-whitelisting)
-Typically you will need to setup an SSH tunnel to access the database you want to
+- `host_name` the host name which can be used to connect to PostgreSQL
+- `postgresql_listen_port` the port to connect to
+- `user_name` the username given in `postgresql_user` combined with the server name in the format postgresql_user@postgres-paas.name
+- `postgresql_database`
+- `postgresql_password` the randomly generated password for the admin login. It will be 16 characters and contain characters from three of the following categories: English uppercase letters, English lowercase letters, numbers (0 through 9), and nonalphanumeric characters (!, $, #, %, etc.).
 
-```
-$ ssh bastion.reform -L 5440:rhubarb-sandbox.postgres.database.azure.com:5432
 
-$ psql -p 5440 -h localhost -U rhubarbadmin@rhubarb-sandbox -d rhubarb
-```
+## Access to databases
+
+Databases are restricted to access from specific subnets, these can be updated in the [cnp-database-subnet-whitelisting](https://github.com/hmcts/cnp-database-subnet-whitelisting)
+GitHub repo.
+
+Typically, you will need to set up an SSH tunnel to access the database you want to.
 
-The password can be retrieved from vault if you've stored it there, or if you view your applications settings.
+All developers can access non production databases with reader access.
 
-Currently developers can only access databases in the sandbox subscription
-The DevOps team can access nonprod databases from the preprod backup box and production databases from the production backup box
+Security cleared developers can access production DBs using just in time access and an approved business justification.
 
-We're currently working on rolling out access via Azure Active Directory, if you've been granted access you can authenticate with:
+_Note: access is only granted on a case by case basis, and is removed automatically_
```

Comment:
It is mentioned about automatic removal. What about the approval process? What will happen once the developer have requested JIT production DB access? Is it approved automatically?

Reply:
No it’s mentioned in here more details to come, that’s being worked out currently.

The proposal is currently the DB owner and someone from senior management have to approve. I’ll probably document that on hmcts.GitHub.io and leave this for the technical bit of connecting